tests/testChebyValues.py

```diff
@@ -87,28 +90,35 @@ def testGetEphemerides(self):
         chebyValues = ChebyValues()
         # chebyValues.setCoefficients(self.chebyFits)
         chebyValues.readCoefficients(self.coeffFile)
+        # Single time, halfway through interval.
         time = self.tStart + self.interval / 2.0
-        # Test for all objects.
+        # Test for a single time, but all the objects.
         ephemerides = chebyValues.getEphemerides(time)
         pyephemerides = self.pyephems.generateEphemerides(time, obscode=807,
                                                           timeScale='TAI', byObject=False)
         # RA and Dec should agree to 2.5mas (skyTolerance above)
-        pos_residuals = np.sqrt((ephemerides['ra'] - pyephemerides['ra'][0]) ** 2 +
-                                (ephemerides['dec'] - pyephemerides['dec'][0]) ** 2)
+        pos_residuals = np.sqrt((ephemerides['ra'][:,0] - pyephemerides['ra'][0]) ** 2 +
```

Comment:
`lsst.sims.utils.angularSeparation()` just does the spherical geometry, if you'd rather use that.